sdk/servicebus/azure-servicebus/tests/test_sessions.py

```diff
@@ -1300,3 +1300,24 @@ def test_session_non_session_send_to_session_queue_should_fail(self, uamqp_trans
                 message = ServiceBusMessage("This should be an invalid non session message")
                 with pytest.raises(ServiceBusError):
                     sender.send_messages(message)
+
+    @pytest.mark.liveTest
+    @pytest.mark.live_test_only
+    @CachedServiceBusResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
+    @ServiceBusQueuePreparer(name_prefix='servicebustest', requires_session=True)
+    @pytest.mark.parametrize("uamqp_transport", uamqp_transport_params, ids=uamqp_transport_ids)
+    @ArgPasser()
+    def test_next_available_session_timeout_value(self, uamqp_transport, *, servicebus_namespace_connection_string=None, servicebus_queue=None, **kwargs):
+        if uamqp_transport==True:
+            pytest.skip("This test is for pyamqp only")
+        with ServiceBusClient.from_connection_string(
+            servicebus_namespace_connection_string, logging_enable=False, uamqp_transport=uamqp_transport, retry_total=1) as sb_client:
+            start_time = time.time()
+            with pytest.raises(OperationTimeoutError):
+                with sb_client.get_queue_receiver(servicebus_queue.name, 
+                                                  session_id=NEXT_AVAILABLE_SESSION, 
+                                                  max_wait_time=10,) as session:
+                    pass
+            end_time = time.time()
```

Comment:
same as above - might be good to test both modes of receiving for consistent behavior